WordPress/Classes/Models/ReaderTeamTopic.swift

```diff
@@ -11,8 +12,8 @@ import Foundation
         return slug == ReaderTeamTopic.a8cSlug ? .readerA8CShown : .readerP2Shown
     }
 
-    var organizationType: SiteOrganizationType {
-        return slug == ReaderTeamTopic.a8cSlug ? .automattic : .p2
+    var organizationType: SiteOrganizationType? {
+        return SiteOrganizationType(rawValue: organizationID.intValue)
```

Comment:
Maybe `return SiteOrganizationType(rawValue: organizationID) ?? .none` to avoid the optional?